modules/machineset-yaml-aws.adoc

```diff
@@ -4,12 +4,24 @@
 // * machine_management/creating_machinesets/creating-machineset-aws.adoc
 // * post_installation_configuration/cluster-tasks.adoc
 
+ifeval::["{context}" == "creating-infrastructure-machinesets"]
+:infra:
+endif::[]
+ifeval::["{context}" == "post-install-cluster-tasks"]
+:infra:
+endif::[]
```

Comment:
"Good idea fairy" sugg: Could you define an attribute that had the value `infra` in those two contexts, and if the attribute was not defined, the value is `<zone>`.  If yes, then the YAML that follows might be a little cleaner with `subs="attributes+"` and the attribute.

Completely up to you.